compiler/src/iree/compiler/Codegen/Common/FlattenMemRefSubspanPass.cpp

```diff
@@ -353,6 +353,15 @@ static Value linearizeIndices(Value sourceValue, ValueRange indices,
   SmallVector<int64_t> strides;
   int64_t offset;
   if (succeeded(getStridesAndOffset(sourceType, strides, offset))) {
+    // For unit dimensions, strides don't matter since the index value can only
```

Comment:
I am not sure this is correct.... if the shape is 1, the stride is same as the stride of the next fastest varying dimension... it isnt 1...